src/client/diagnostics.rs

```diff
@@ -0,0 +1,133 @@
+use bevy::diagnostic::DiagnosticId;
+use bevy::{
+    diagnostic::{Diagnostic, Diagnostics, RegisterDiagnostic},
+    prelude::*,
+    time::common_conditions::on_timer,
+};
+use std::time::Duration;
+
+/// Replication stats during packet processing.
+///
+/// Flushed to Diagnostics system periodically.
+#[derive(Default, Resource, Debug)]
+pub struct ReplicationStats {
+    /// Incremented per entity that changes.
+    pub entities_changed: u32,
```

Comment:
Maybe it worth to use `u64` for all stats?
Looks like Bevy usually use 8 bytes for diagnostic data.